tests/jax/test_praxis_layers.py

```diff
@@ -40,6 +43,19 @@
 FP8_FORMATS = [Format.E4M3, Format.HYBRID]
 
 
+@pytest.fixture(autouse=True, scope='module')
+def enable_fused_attn():
+    """
+    Enable fused attn for hopper+ arch.
+    Fused attn kernels on pre-hopper arch are not deterministic.
```

Comment:
I think this might be wrong here. For pre-hopper archs, kernels are deterministic, and for hopper+, the workspace_optimization path gives deterministic answers. This is for arbitrary seqlen backend. The max512 backend is always deterministic.